python-sdk/src/langwatch/openai.py

```
@@ -550,16 +550,82 @@ def handle_deltas(
+                            )
+                            if delta.function_call.name:
+                                last_item["function_call"]["name"] = (
+                                    delta.function_call.name
+                                )
+                        else:
+                            last_item["function_call"] = {
+                                "name": delta.function_call.name or "",
+                                "arguments": delta.function_call.arguments or "",
+                            }
+                    if delta.tool_calls:
+                        if "tool_calls" in last_item and last_item["tool_calls"]:
```

Ruff (0.15.21)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)